gen/calc.py

```diff
@@ -492,7 +566,6 @@ def validate_exhibitor_storage_master_discovery(master_discovery, exhibitor_stor
         'master_dns_bindall': 'true',
         'mesos_dns_ip_sources': '["host", "netinfo"]',
         'master_external_loadbalancer': '',
-        'mesos_container_logger': __logrotate_slave_module_name,
```

Comment:
Should also remove the definition of that `__logrotate_slave_module_name` variable
https://github.com/dcos/dcos/pull/1038/files#diff-3cd4ddcecc89d0887dd9cb3d5fabe995R511